docs/edgeql/select.rst

```diff
@@ -843,6 +843,48 @@ shapes just like a non-computed link.
   } filter .name = "Iron Man";
 
 
+Using backlinks to find items
+-----------------------------
+
+Another handy use for backlinks is using them to filter and find items
+such as during an update. This can work as a nice shortcut when you have
+the id of one object that links to a second object without a link back
+to the first.
+
+Spider-Man's villains always have a grudging respect for him, and their names
+can be changed to reflect that if we know the id of a movie that they starred
+in. (Note the ability to :ref:`cast <ref_eql_casts>` from a ``uuid`` to an
+object type, which was added in EdgeDB 3.0!)
+
+.. code-block:: edgeql-repl
+    
+    db> update Villain filter .<characters = 
+    ...   <Movie><uuid>'6c60c28a-5c03-11ee-99ff-dfa425012a05'
+    ...   set { 
+    ...     name := .name ++ ', who got to see Spider-Man!' 
+    ...   };
+
+    db> select Villain.name;
+      {
+        'Obadiah Stane',
+        'Sandman, who got to see Spider-Man!',
+        'Electro, who got to see Spider-Man!',
+        'Green Goblin, who got to see Spider-Man!',
+        'Doc Ock, who got to see Spider-Man!',
+      }
+
+In other words, "update every ``Villain`` object that the ``Movie`` object
+of this id links to via a link called ``characters``".
+
+A backlink is naturally not required, however. The same update without
+traversing a backlink would look like this:
+
```

Comment:
```suggestion

.. code-block:: edgeql-repl

```

Before you push your final changes, run the site dev server to make sure they build locally. Let me know if you're having trouble building it generally. We can try to figure out the issue and make sure it builds for you.